### ISSUE TITLE
Incorrect implicit_layer.d files

### ISSUE THREAD

**Issue by hitman249** [flightlessmango/MangoHud#221]
https://github.com/flightlessmango/MangoHud/blob/master/src/mangohud.json.in#L4

Currents:
```json
"name": "MangoHud x86",
"name": "MangoHud x86_64",
```

Corrects:
```json
"name":  "VK_LAYER_MangoHud_32",
"name":  "VK_LAYER_MangoHud_64",
```

Problem:

```bash
env VK_INSTANCE_LAYERS="$VK_INSTANCE_LAYERS:MangoHud x86:MangoHud x86_64" ...
# error, spaces are not allowed
```

**Comment by jackun** [comment]
"Explicit" but officially it should even be something like  `VK_LAYER_MANGOHUD_hud32`

**Comment by jackun** [comment]
Should be fixed in d712d35aaf5d548f23cc1e1596646cbafc4004a3
